app/assets/javascripts/modules/image_x_viewer.js

```diff
@@ -253,7 +253,10 @@
     var _setupButtonListeners = function() {
       $embedHeader.find('[data-sul-view-fullscreen="fullscreen"]')
         .on('click', function() {
+          var worldRestriction = $('[data-world-restriction]').data('world-restriction');
           canvasStore.osd.setFullScreen(true);
+          $('.osd-container').attr('data-world-restriction', worldRestriction);
```

Comment:
The `world-restriction` data attribute is not present on the full-screen OSD viewer so we need to add it there so `_disableClicks()` knows to disable the right-click.